dashboard/db/schema.rb

```diff
@@ -81,6 +81,17 @@
     t.index ["user_id", "level_id", "script_id"], name: "index_ace_user_level_script"
   end
 
+  create_table "aichat_requests", charset: "utf8mb4", collation: "utf8mb4_bin", force: :cascade do |t|
+    t.integer "user_id", null: false
```

Comment:
It doesn't seem like it matters but just curious - is there a reason you're using strings instead of symbols for the table column names?

Reply:
Hmm do you mean like `t.integer :user_id` instead of `"user_id"`? This was autogenerated by the rails create model command. I just edited it to add `null: false` on a few columns.

Reply:
Oh gotcha! Actually I think I thought I was looking at the migration file when I wrote the comment. Thanks!